examples/simple/e2e.yaml

```diff
@@ -32,3 +34,13 @@ setup:
           resource: pod
           for: condition=Ready
   timeout: 600
+
+verify:
+  - actual: ../../test/verify/1.actual.yaml
```

Comment:
Whether actual data is in memory or in the static yaml? Or this yaml is generated?

Reply:
> Whether actual data is in memory or in the static yaml? Or this yaml is generated?

This example `e2e.yaml` demonstrates 2 scenarios, 1) actual data is generated in users tests, (your question 👆), 2) actual data is in-memory via query command, both are obviously not static

```yaml
- actual: ../../test/verify/1.actual.yaml  # scenario 1
  expected: ../../test/verify/2.expected.yaml
- query: swctl --display yaml service ls   # scenario 2
  expected: ../../test/verify/3.expected.yaml
```